corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -101,46 +101,45 @@ def process_bulk_app_translation_upload(app, workbook, expected_headers, lang=No
             msgs.extend(_process_rows(app, MODULES_AND_FORMS_SHEET_NAME,
                                       modules_and_forms_rows, lang=lang))
         else:
-            msgs.extend(_process_rows(app, sheet.worksheet.title, sheet,
-                                      sheet_name=sheet.worksheet.title))
+            msgs.extend(_process_rows(app, sheet.worksheet.title, sheet))
 
     msgs.append(
         (messages.success, _("App Translations Updated!"))
     )
     return msgs
 
 
-def _process_rows(app, identifier, rows, sheet_name=None, lang=None):
```

Comment:
Nice cleanup, this logic is definitely clearer now.

Reply:
Good call. Some comments and a docstring would make this much more readable.